html/index.html

```diff
@@ -40,6 +40,10 @@
                     <li><button id="button-player-more" class="button button--sacnite button--inverted" alt="More players" title="More players"><i class="icon-user-plus"></i><span>+</span></button></li>
                     <li><button id="button-player-less" class="button button--sacnite button--inverted" alt="Less players" title="Less players"><i class="icon-user-less"></i><span>-</span></button></li>
                 </ul>
+                <ul class="vertical-centered">
+                    <li><button id="button-merge" class="button button--sacnite button--inverted" alt="Merge isets"><i class="icon-link"></i><span>+</span></button></li>
```

Comment:
I think you can delete the <span>+</span> in this button and the next. Maybe also in all the other buttons?